powerline/segments/i3wm.py

```diff
@@ -54,12 +54,44 @@ def workspaces(pl, segment_info, only_show=None, output=None, strip=0):
 		'contents': w['name'][min(len(w['name']), strip):],
```

Comment:
By the way, what was this `min()` call for originally? `''[10:] == ''`, even though `len('') == 0`.